app/javascript/dashboard/assets/scss/widgets/_conv-header.scss

```diff
@@ -56,6 +56,19 @@ $resolve-button-width: 13.2rem;
       flex-direction: column;
       justify-content: center;
       margin-left: $space-slab;
+      max-width: 50rem;
+
+      @media screen and (max-width: 1600px) {
```

Comment:
We have mixins for responsive sizes. Please see this file `components/widgets/conversation/ConversationBox.vue`.